scalingo/main.go

```diff
@@ -89,7 +89,8 @@ COMMANDS:{{range .VisibleCategories}}{{if .Name}}
 		 {{end}}{{end}}{{if .VisibleFlags}}
 GLOBAL OPTIONS:
    {{range $index, $option := .VisibleFlags}}{{if $index}}
-   {{end}}{{$option}}{{end}}{{end}}{{if .Copyright}}
+   {{end}}{{$option}}{{end}}{{end}}
+{{if .Copyright}}
```

Comment:
Is it valid to put the `if` statement on a new line?

Reply:
Yes we're in a template and I wanted to add a newline because the output was not great (really compacted)